io.github.Youda008.DoomRunner.appdata.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.Youda008.DoomRunner</id>
+  <launchable type="desktop-id">io.github.Youda008.DoomRunner.desktop</launchable>
+  <name>Doom Runner</name>
+  <summary>Preset-oriented graphical launcher of ZDoom and derivatives</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-only</project_license>
+  <developer_name>Youda008</developer_name>
+  <url type="homepage">https://github.com/Youda008/DoomRunner</url>
+  <url type="bugtracker">https://github.com/Youda008/DoomRunner/issues</url>
+  <url type="help">https://github.com/Youda008/DoomRunner#readme</url>
+  <description>
+    <p>
+      Doom Runner is yet another launcher of common Doom source ports (like GZDoom, Zandronum, PrBoom, ...) with graphical user interface.
+      It is written in C++ and Qt, and it is designed around the idea of presets for various multi-file modifications (Brutal Doom with
```

Comment:
I don't see any user caring about `C++` or `Qt`. what's important is what this does, and eventually anything that helps running it like the tedious game file installation.

Reply:
The author already accepted the [PR #127](https://github.com/Youda008/DoomRunner/pull/127) to add the `.appdata.xml` file upstream. As soon the next version will be released, I'll remove the file from Flathub repo and I'll include it from sources.
Instructions will remain on README to help users to manage files and Flatpak permissions.

Reply:
BTW, if you know "less tedious" ways to use game files or external engines, please let us know. Right now I didn't find a better way to use a launcher from a sandbox, but maybe it's not the best way. Also run sandboxed engines is not straightforward.